Makefile

```
@@ -11,6 +12,7 @@ MANDIR ?= $(DESTDIR)$(PREFIX)/share/man/man1
 INSTALL = install
 
 $(TARGET): $(OBJS)
+	$(CC) $(CFLAGS) $(OBJS) -o $(TARGET) $(LDLIBS)
```

This should not be using `CFLAGS`
```suggestion
	$(CC)  $(LDLIBS) $(OBJS) -o $(TARGET)
```